practice-project-bryce/src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -0,0 +1,81 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import com.revrobotics.CANSparkBase.IdleMode;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Elevator extends SubsystemBase {
+    private CANSparkMax m_leftElevatorMotor;
+    private SparkPIDController m_leftPidController;
+    private RelativeEncoder m_leftEncoder;
+
+    private CANSparkMax m_rightElevatorMotor;
+
+    private int smartMotionSlot = 0;
+
+    public Elevator(){
+        m_leftElevatorMotor = new CANSparkMax(25, CANSparkLowLevel.MotorType.kBrushless);
+        m_leftPidController = m_leftElevatorMotor.getPIDController();
+        m_leftEncoder = m_leftElevatorMotor.getEncoder();
+
+        m_rightElevatorMotor = new CANSparkMax(31, CANSparkLowLevel.MotorType.kBrushless);
+        
+        configure();
+    }
+
+    public void configure(){
+        m_leftElevatorMotor.setInverted(false);
```

Comment:
All of these values you are setting into the config should be declared in Constants.java

Reply:
done